lib/ruby_terraform/commands/import.rb

```diff
@@ -1,45 +1,37 @@
 # frozen_string_literal: true
 
-require 'json'
 require_relative 'base'
+require_relative '../command_line/options'
 
 module RubyTerraform
   module Commands
-    class Import < Base
-      def configure_command(builder, opts)
-        directory = opts[:directory]
-        vars = opts[:vars] || {}
-        var_file = opts[:var_file]
-        var_files = opts[:var_files] || []
-        no_color = opts[:no_color]
-        no_backup = opts[:no_backup]
-        backup = no_backup ? '-' : opts[:backup]
-        state = opts[:state]
-        input = opts[:input]
-        address = opts[:address]
-        id = opts[:id]
+    class Import < Base#
```

Comment:
What's the `#` here for?

Reply:
quick way to make guard fire the tests on the current class - make an inconsequential change to it - then forget you did it and commit it.  :-)